### PR TITLE
build: bump dev dependencies

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Bump most (except TS and eslint) dev dependencies to inherits most recent tooling improvements.

# Changes

- Bump latest dev dependencies

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,10 @@
 - Added `memoToNeuronSubaccount` and `memoToNeuronAccountIdentifier`.
 - Support new neuron field `voting_power_refreshed_timestamp_seconds`.
 
+## Build
+
+- Update the dev dependencies of the `ic-js` monorepo.
```
Comment (dskloetd) [inline CHANGELOG.md:14]: Why do you call it the monorepo?

**Review by dskloetd — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (peterpeterparker) [reply CHANGELOG.md:14]: I wanted to highlight the fact that ic-js itself is not a library per sé. I can use the term "workspaces" if you rather like?

Comment (peterpeterparker) [reply CHANGELOG.md]: Updated to "workspace" to move on and merge the PR.

Comment (dskloetd) [reply CHANGELOG.md]: Yes, "workspace" makes sense.
To me a monorepo encompasses much more. The fact that nns-dapp and ic-js aren't even in the same repo makes it definitely not a monorepo. But I don't know if my understanding of "monorepo" is "correct".

**Comment by peterpeterparker** [comment]
@dskloetd can you review / approve again? Looks like merging main and update the text requires a new review now.

**Review by dskloetd — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (peterpeterparker) [reply CHANGELOG.md]: It's probably a fine grain here, probably depends if we consider our libs as projects or not. Anyway, for sure I'm fine with the change, thanks for the feedback.

**Pull request merged**